python/ray/train/xgboost/v2.py

```diff
@@ -53,17 +51,16 @@ def train_fn_per_worker(config: dict):
                 "max_depth": 2,
             }
 
-            # 2. Do distributed data-parallel training with the `CommunicatorContext`.
+            # 2. Do distributed data-parallel training.
             # Ray Train sets up the necessary coordinator processes and
             # environment variables for your workers to communicate with each other.
-            with CommunicatorContext():
```

Comment:
What would happen when using a single worker for training? Will the context manager here be a noop?

Reply:
Yeah, it works with single worker